packages/app/src/files-manager/helpers/display-space.tsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const displaySpace = (space: number): string => {
+  const value =
+    space >= 1000 ? `${(space / 1000).toFixed(1)} GB` : `${space} MB`
+  return value
+}
```

Comment:
What with other sizes, like "B", "KB"?

The space is a megabyte as default? Maybe refactore space name to `megabyte`, `megabyteSize` or `megabyteNumber`? 🤔 

`convertBytes` helper could be useful here?

Reply:
Here space should be in `mebibyte`. Refactor for display-space in [495ae4d](https://github.com/mudita/mudita-center/pull/598/commits/495ae4d3bfc3376fabc34ee880f1915c53df88b9).